src/core/player.h

```diff
@@ -119,6 +120,14 @@ class Player : public PlayerInterface {
   explicit Player(Application* app, QObject* parent = nullptr);
   ~Player();
 
+  static const char* kSettingsGroup;
+
+  // Don't change the values
```

Comment:
because they are stored in the settings, right?
Maybe better to write this down, to make things really clear and meaningful FWIW